docs/requirements.txt

```diff
@@ -6,6 +6,9 @@ docutils>=0.16
 sphinx-paramlinks>=0.4.0
 ipython[notebook]
 
+# temporarily included until hub available to evaluate finetuning_scheduler
+git+git://github.com/speediedan/pytorch-lightning.git@24d3e43568814ec381ac5be91627629808d62081#egg=pytorch-lightning
```

Comment:
this shall be in meta requirements

Reply:
Thanks for taking a look so quickly (and the awesome tutorials work!) I originally added to .meta.yml but was getting a ``KeyError`` for the "git*" package reference  in [meta["environment"] = [env[r] for r in require]](https://github.com/speediedan/lightning-tutorials/blob/898b8b6f9852c0b80f034a3187bc1cd34dd521ce/.actions/helpers.py#L474) so included it in docs/requirements.txt just as a temporary workaround (since the dependency will disappear altogether when it is registered via the new api I think). 

Since it's only temporary (until the relevant module is pulled from the hub), I've gone ahead and removed the pytorch_lightning fork from the requirements and everything should work if one manually installs the relevant pytorch-lightning fork for testing i.e.
```bash
pip install git+git://github.com/speediedan/pytorch-lightning.git@24d3e43568814ec381ac5be91627629808d62081#egg=pytorch-lightning
```
Let me know if you know and prefer a way around the KeyError for 'git*' package references in helpers.py instead. Thanks again!

Reply:
that is a very good point, could you please open an issue to add this support?

Reply:
Done! #116

Reply:
shall be fixed now on master :]